_data/Panther2804/Panther2804.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Panther2804
+institution: Willhelm-Hausenstein-Gymnasium
```

Comment:
Possible typo with a double _ll_? Change Willhelm-Hausenstein-Gymnasium to Wilhelm-Hausenstein-Gymnasium?